packages/flutter/test/cupertino/route_test.dart

```diff
@@ -1471,6 +1472,26 @@ void main() {
     expect(find.text('first', skipOffstage: false), findsOneWidget);
     expect(find.text('second'), findsOneWidget);
   });
+
+  testWidgets('Popping routes should cancel down events', (WidgetTester tester) async {
```

Comment:
I've added the following comment, is it clearer?
```dart
    // The test target, _TestPostRouteCancel, shows on the home route how many
    // [PointerCancelEvent] it has received. Initially it will show "Home 0".
    // After the route is popped, it should show "Home 1", meaning it received
    // one [PointerCancelEvent].
```